packages/store/src/public_api.ts

```diff
@@ -33,6 +33,9 @@ export {
 export {
   StateContext,
   StateOperator,
+  ExistingState,
```

Comment:
I think this should be exported with a barred O indicating it's a private stuff.

Reply:
This still can be used by library consumers who work on custom operators in their apps. But still should be exported as private symbols to indicate that the type might be ch aged between versions and use it on their risk.

Reply:
I do consider this type as part of the public api, because people should be able to create their own operators. 
I'm thinking that there needs to be a operator types specific subpackage though

Reply:
Well, I consider this to be a sub package so it’s not tied to the core. Also that would remove a reference from operators of the core package.

Reply:
Umm, well… lemme get to PC and check again

Reply:
It could just be a types import into the core package from the operators subpackage

Reply:
I'm ok with leaving it in the core